v3/examples/dock/build/linux/nfpm/nfpm.yaml

```
@@ -0,0 +1,73 @@
+  - pkg-config
+
+# Distribution-specific overrides for different package formats and WebKit versions
+overrides:
+  # RPM packages for RHEL/CentOS/AlmaLinux/Rocky Linux (WebKit 4.0)
+  rpm:
+    depends:
+      - gtk3-devel
+      - webkit2gtk3-devel
+      - gcc-c++
+      - pkg-config
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)